ledger/acctupdates.go

```diff
@@ -291,6 +347,10 @@ func (au *accountUpdates) committedUpTo(rnd basics.Round) basics.Round {
 				return err
 			}
 
+			for aidx := range au.assetDeltas[i] {
```

Comment:
Reset `assetFlushcount` at the top of this `Atomic()` like `flushcount`.

Reply:
added, oops! since this would have panicked and was not caught by tests, we should definitely add a test that actually sends txn and waits more than lookback rounds...